hydropandas/extensions/plots.py

```diff
@@ -307,6 +307,247 @@ def interactive_map(
             m.save(os.path.join(plot_dir, fname))
 
         return m
+    
+    def section_plot(
+        self, tmin=None, tmax=None,
+        cols=(None,),
+        section_colname_x=None,
+        section_label_x=None,
+        section_well_layout_color='gray',
+        section_markersize=100,
+        ylabel='auto',
+        fn_save=None,
+        check_obs_close_to_screen_bottom=True,
+        ):
+        
+        """Create plot with well layout (left) en observations (right).
+
+        Parameters
+        ----------
+        tmin : dt.datetime, optional
+            start date for timeseries plot
+        tmax : dt.datetime, optional
+            end date for timeseries plot
+        cols : tuple of str or None, optional
+                the columns of the observation to plot. The first numeric column
+                is used if cols is None, by default None.
+        section_colname_x : str, optional
+            column used for x position on section plot, when None order collection is used
+        section_label_x : str, optional
+            label applied to x-axis in section plot
+        section_well_layout_color : str, optional
+            color of well layout, default is gray
+        section_markersize : int, optional
+            size of makers in sectionplot
+        ylabel: str or list, optional
+            when 'auto' column unit in collection is ylabel, otherwise first
+            element of list is label of section plot, second element of observation plot
+        fn_save : str, optional
+            filename to save plot
+        check_obs_close_to_screen_bottom : bool, optional
+            plots a horizontal line when minimum observation is close to screen_bottom
+
+        TODO:
+            - speficy colors via extra column in ObsCollection
+            - addtional visual checks:
+                maximum observation is close to or above ground level,
+                maximum observation is close to or above tube top
+                minimum observation is close to or below tube bottom (sand trap)
+            - include some interactive Bokeh fancy
+            - apply an offset when two wells are at same location
+            - limit y-axis of section plot to observations only
+        """
+        
+        import matplotlib.pyplot as plt
```

Comment:
These can be moved to the top of the file imo.

Reply:
Solved in update.